test/Analysis/ctu-on-the-fly.c

```diff
@@ -0,0 +1,75 @@
+// RUN: rm -rf %t && mkdir %t
+// RUN: mkdir -p %t/ctudir2
+// RUN: cp %S/Inputs/ctu-other.c %t/ctudir2/ctu-other.c
+// RUN: echo '[{"directory":"%t/ctudir2","command":"clang -c ctu-other.c","file":"ctu-other.c"}]' | sed -e 's/\\/\\\\/g' > %t/ctudir2/compile_commands.json
```

Comment:
We cannot, because the compile commands json should contain paths, which can only be generated of the actual filesystem of the test-runner. See updated tests ctu-on-demand-parsing.c